react-common/components/controls/FocusList.tsx

```diff
@@ -57,42 +59,6 @@ export const FocusList = (props: FocusListProps) => {
         }
     }
 
-    const isFocusable = (e: HTMLElement) => {
-        return e.getAttribute("data-isfocusable") === "true"
```

Comment:
the data-isfocusable attribute still needs to be checked for the focuslist. Maybe findNextFocusableElement should optionally take in a predicate function?

Reply:
This is something that I didn't fully grasp. In `handleRef` where the `focusableElements` list is populated, the query selector looks for `data-isfocusable`. Are both really needed?

Reply:
data-isfocusable can be set to false programatically. for example, in Tree.tsx

Reply:
Gotcha, I'll add it back.